recipes/user_classes_design_recipe.md

```diff
@@ -0,0 +1,161 @@
+# {{TABLE NAME}} Model and Repository Classes Design Recipe
```

Comment:
Previously I'd have done this in the same document as the peep, but i prefer this way, keeps it clearer